client/state/stats/lists/utils.js

```diff
@@ -529,6 +529,34 @@ export const normalizers = {
 		return { total_wpcom, total_email, total, subscribers };
 	},
 
+	statsUTM( originalData ) {
```

Comment:
This is picked up from the custom data normalizer we have for the UTM stats download - https://github.com/Automattic/wp-calypso/blob/e93071be9020b4cf11eb9096b9faf89acf3c25bf/client/my-sites/stats/features/modules/stats-utm/utm-export-button.jsx#L7-L32